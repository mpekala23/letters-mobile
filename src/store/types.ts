import { UserState } from "./User/UserTypes";
import { NotifState } from "./Notif/NotifTypes";
import { ContactState } from "./Contact/ContactTypes";

export interface AppState {
  user: UserState;
  notif: NotifState;
  contact: ContactState;
}
