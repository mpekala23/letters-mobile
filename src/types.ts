// TODO: make this typing better
// probably want a specific type for state abbrevs
export interface Facility {
  name: string;
  type: PrisonTypes;
  address: string;
  city: string;
  state: string;
  postal: string;
  phone: string;
}

export enum ProfilePicTypes {
  Topbar = 'Topbar',
  Contact = 'Contact',
  SingleContact = 'SingleContact',
}

export enum Storage {
  RememberToken = 'Ameelio-Token',
}

export enum PrisonTypes {
  State = 'State',
  Federal = 'Federal',
  Jail = 'County',
}

export enum SupportFAQTypes {
  DeleteLetter = 'DeleteLetter',
  NotArrived = 'NotArrived',
  WrongReturnAddress = 'WrongReturnAddress',
  WrongMailingAddress = 'WrongMailingAddress',
  TrackingNumber = 'TrackingNumber',
  TrackingError = 'TrackingError',
  TalkToAmeelio = 'TalkToAmeelio',
}

export enum LetterStatus {
  Draft = 'Draft',
  Created = 'Created',
  Mailed = 'Mailed',
  InTransit = 'In Transit',
  InLocalArea = 'In Local Area',
  OutForDelivery = 'Out for Delivery',
  Delivered = 'Delivered',
}

export enum LetterTypes {
  Postcard = 'postcard',
  Letter = 'letter',
}

export enum DeliveryReportTypes {
  received = 'received',
  unsure = 'unsure',
  notYetReceived = 'notYetReceived',
  haveNotAsked = 'haveNotAsked',
  haveNotReceived = 'haveNotReceived',
}

export interface Letter {
  type: LetterTypes;
  status: LetterStatus;
  isDraft: boolean;
  recipientId: number;
  content: string;
  photo?: Photo;
  letterId?: number; // TODO: Once we have more info on this field and lob, use this more
  expectedDeliveryDate?: Date;
  dateCreated?: Date;
  trackingEvents?: LetterTrackingEvent[];
}

export interface LetterTrackingEvent {
  id: number;
  name: string;
  location: ZipcodeInfo;
  date: Date;
}

export type TopbarRouteAction = {
  enabled: boolean;
  text: string;
  action: () => void | Promise<void>;
  blocking?: boolean;
};

export interface ZipcodeInfo {
  zip: string;
  city: string;
  state: string;
  lat?: number;
  long?: number;
}

export interface Photo {
  type?: 'image' | string;
  width?: number;
  height?: number;
  uri: string;
}
