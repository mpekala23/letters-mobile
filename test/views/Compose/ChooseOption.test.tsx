import React from 'react';
import { ChooseOptionScreen } from '@views';
import { render, fireEvent, toJSON } from '@testing-library/react-native';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import { LetterTypes } from 'types';

const mockStore = configureStore([]);

const setup = () => {
  const navigation = { navigate: jest.fn() };
  const store = mockStore({
    letter: {
      composing: {
        type: LetterTypes.Postcard,
        recipient: null,
        content: '',
      },
      existing: [],
    },
    user: {
      user: {
        address1: 'Somewhere',
        address2: 'Apt A',
        city: 'Springfield',
        state: 'Illinois',
        postal: '90210',
      },
    },
    contact: {
      active: {
        id: 1,
      },
    },
  });
  const StoreProvider = ({ children }: { children: JSX.Element }) => {
    return <Provider store={store}>{children}</Provider>;
  };

  return {
    navigation,
    store,
    ...render(<ChooseOptionScreen navigation={navigation} />, {
      wrapper: StoreProvider,
    }),
  };
};

describe('ChooseOption screen', () => {
  it('should match snapshot', () => {
    const { container } = setup();
    const tree = toJSON(container);
    expect(tree).toMatchSnapshot();
  });

  it('should dispatch a setType action when Post cards button is pressed', () => {
    const { store, getByText } = setup();
    fireEvent.press(getByText('Post cards'));
    const actions = store.getActions();
    expect(actions.length).toBe(2);
    expect(actions[0].type).toBe('letter/set_type');
    expect(actions[0].payload).toBe('postcard');
  });

  it('should dispatch a setType action when Letter button is pressed', () => {
    const { store, getByText } = setup();
    fireEvent.press(getByText('Letters'));
    const actions = store.getActions();
    expect(actions.length).toBe(2);
    expect(actions[0].type).toBe('letter/set_type');
    expect(actions[0].payload).toBe('letter');
  });
});
