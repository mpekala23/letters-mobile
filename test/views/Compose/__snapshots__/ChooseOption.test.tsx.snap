// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChooseOption screen should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "flex": 1,
        "flexDirection": "column",
        "padding": 16,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "Poppins-Bold",
          },
          Object {
            "fontSize": 22,
            "marginBottom": 4,
            "marginTop": 10,
          },
        ]
      }
    >
      Choose an option
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontFamily": "Poppins-Regular",
          },
          Object {
            "color": "#6D6D6D",
            "fontSize": 14,
            "paddingBottom": 10,
          },
        ]
      }
    >
      P.S. Your loved ones will be able to respond to you via the return address:
       
      Somewhere
       Apt A
      , 
      Springfield
      ,
       
      IL
       
      90210
      .
    </Text>
    <TouchableOpacity
      activeOpacity={0.2}
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderRadius": 6,
            "marginVertical": 6,
            "padding": 15,
            "width": "100%",
          },
          Object {
            "flexDirection": "column",
            "height": 110,
            "justifyContent": "space-between",
          },
          Object {
            "elevation": 4,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 4,
              "width": 0,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 2.62,
          },
          undefined,
        ]
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight={68}
        bbWidth={111}
        fill="none"
        focusable={false}
        height={68}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "bottom": 0,
              "position": "absolute",
              "right": 0,
            },
            Object {
              "flex": 0,
              "height": 68,
              "width": 111,
            },
          ]
        }
        vbHeight={68}
        vbWidth={111}
        width={111}
        xml="<svg width=\\"111\\" height=\\"68\\" viewBox=\\"0 0 111 68\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\">
  <path
    d=\\"M54.9922 13.5328L78.4832 18.8756L54.9922 13.5328ZM52.855 22.9292L62.2514 25.0663L52.855 22.9292ZM60.3263 44.3934L53.067 32.8599L46.0197 31.257C44.7737 30.9736 43.6913 30.2068 43.0106 29.1254C42.3299 28.0439 42.1067 26.7363 42.3901 25.4903L46.6643 6.69746C46.9477 5.45143 47.7145 4.369 48.796 3.68831C49.8775 3.00762 51.1851 2.78443 52.4311 3.06783L85.3185 10.5478C86.5645 10.8312 87.6469 11.598 88.3276 12.6795C89.0083 13.7609 89.2315 15.0685 88.9481 16.3146L84.6739 35.1073C84.3905 36.3534 83.6237 37.4358 82.5422 38.1165C81.4607 38.7972 80.1531 39.0204 78.9071 38.737L71.8598 37.1341L60.3263 44.3934Z\\"
    stroke=\\"#448AF3\\"
    strokeWidth=\\"3\\"
    strokeLinecap=\\"round\\"
    strokeLinejoin=\\"round\\"
  />
  <path
    d=\\"M10.5821 49.9328L23.9331 42.1375L10.5821 49.9328ZM13.7002 55.2732L19.0406 52.1551L13.7002 55.2732ZM26.6119 62.0565L18.1534 59.8341L14.148 62.1727C13.4399 62.5862 12.5964 62.7014 11.8033 62.493C11.0101 62.2846 10.3323 61.7697 9.91878 61.0615L3.68258 50.3807C3.2691 49.6725 3.15387 48.829 3.36226 48.0359C3.57064 47.2428 4.08556 46.5649 4.79375 46.1514L23.4853 35.2381C24.1934 34.8246 25.0369 34.7094 25.83 34.9177C26.6232 35.1261 27.301 35.641 27.7145 36.3492L33.9507 47.0301C34.3642 47.7383 34.4794 48.5817 34.271 49.3748C34.0627 50.168 33.5477 50.8459 32.8395 51.2594L28.8342 53.5979L26.6119 62.0565Z\\"
    stroke=\\"#448AF3\\"
    strokeWidth=\\"3\\"
    strokeLinecap=\\"round\\"
    strokeLinejoin=\\"round\\"
  />
  <path
    d=\\"M92.2365 57.7015L105.219 57.5161L92.2365 57.7015ZM92.3107 62.8947L97.5038 62.8205L92.3107 62.8947ZM98.9505 73.1882L93.6832 68.0692L89.7883 68.1249C89.0997 68.1347 88.4353 67.8706 87.9414 67.3906C87.4475 66.9106 87.1645 66.254 87.1547 65.5654L87.0063 55.1791C86.9965 54.4905 87.2606 53.8261 87.7406 53.3322C88.2206 52.8383 88.8771 52.5553 89.5658 52.5455L107.742 52.2859C108.43 52.2761 109.095 52.5402 109.589 53.0202C110.083 53.5002 110.366 54.1567 110.375 54.8454L110.524 65.2316C110.534 65.9203 110.269 66.5846 109.789 67.0785C109.309 67.5724 108.653 67.8554 107.964 67.8653L104.069 67.9209L98.9505 73.1882Z\\"
    stroke=\\"#448AF3\\"
    strokeWidth=\\"3\\"
    strokeLinecap=\\"round\\"
    strokeLinejoin=\\"round\\"
  />
</svg>"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            Array [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M54.9922 13.5328L78.4832 18.8756L54.9922 13.5328ZM52.855 22.9292L62.2514 25.0663L52.855 22.9292ZM60.3263 44.3934L53.067 32.8599L46.0197 31.257C44.7737 30.9736 43.6913 30.2068 43.0106 29.1254C42.3299 28.0439 42.1067 26.7363 42.3901 25.4903L46.6643 6.69746C46.9477 5.45143 47.7145 4.369 48.796 3.68831C49.8775 3.00762 51.1851 2.78443 52.4311 3.06783L85.3185 10.5478C86.5645 10.8312 87.6469 11.598 88.3276 12.6795C89.0083 13.7609 89.2315 15.0685 88.9481 16.3146L84.6739 35.1073C84.3905 36.3534 83.6237 37.4358 82.5422 38.1165C81.4607 38.7972 80.1531 39.0204 78.9071 38.737L71.8598 37.1341L60.3263 44.3934Z"
            propList={
              Array [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={4282682099}
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth={3}
          />
          <RNSVGPath
            d="M10.5821 49.9328L23.9331 42.1375L10.5821 49.9328ZM13.7002 55.2732L19.0406 52.1551L13.7002 55.2732ZM26.6119 62.0565L18.1534 59.8341L14.148 62.1727C13.4399 62.5862 12.5964 62.7014 11.8033 62.493C11.0101 62.2846 10.3323 61.7697 9.91878 61.0615L3.68258 50.3807C3.2691 49.6725 3.15387 48.829 3.36226 48.0359C3.57064 47.2428 4.08556 46.5649 4.79375 46.1514L23.4853 35.2381C24.1934 34.8246 25.0369 34.7094 25.83 34.9177C26.6232 35.1261 27.301 35.641 27.7145 36.3492L33.9507 47.0301C34.3642 47.7383 34.4794 48.5817 34.271 49.3748C34.0627 50.168 33.5477 50.8459 32.8395 51.2594L28.8342 53.5979L26.6119 62.0565Z"
            propList={
              Array [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={4282682099}
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth={3}
          />
          <RNSVGPath
            d="M92.2365 57.7015L105.219 57.5161L92.2365 57.7015ZM92.3107 62.8947L97.5038 62.8205L92.3107 62.8947ZM98.9505 73.1882L93.6832 68.0692L89.7883 68.1249C89.0997 68.1347 88.4353 67.8706 87.9414 67.3906C87.4475 66.9106 87.1645 66.254 87.1547 65.5654L87.0063 55.1791C86.9965 54.4905 87.2606 53.8261 87.7406 53.3322C88.2206 52.8383 88.8771 52.5553 89.5658 52.5455L107.742 52.2859C108.43 52.2761 109.095 52.5402 109.589 53.0202C110.083 53.5002 110.366 54.1567 110.375 54.8454L110.524 65.2316C110.534 65.9203 110.269 66.5846 109.789 67.0785C109.309 67.5724 108.653 67.8554 107.964 67.8653L104.069 67.9209L98.9505 73.1882Z"
            propList={
              Array [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={4282682099}
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth={3}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Poppins-Bold",
            },
            Object {
              "color": "#282828",
              "fontSize": 24,
            },
          ]
        }
      >
        Photos
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#6E757C",
              "fontSize": 18,
            },
            Object {
              "maxWidth": "60%",
            },
          ]
        }
      >
        Glossy photo & short caption
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderRadius": 6,
            "marginVertical": 6,
            "padding": 15,
            "width": "100%",
          },
          Object {
            "flexDirection": "column",
            "height": 110,
            "justifyContent": "space-between",
          },
          Object {
            "elevation": 4,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 4,
              "width": 0,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 2.62,
          },
          undefined,
        ]
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight={67}
        bbWidth={115}
        fill="none"
        focusable={false}
        height={67}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "bottom": 0,
              "position": "absolute",
              "right": 0,
            },
            Object {
              "flex": 0,
              "height": 67,
              "width": 115,
            },
          ]
        }
        vbHeight={67}
        vbWidth={115}
        width={115}
        xml="<svg width=\\"115\\" height=\\"67\\" viewBox=\\"0 0 115 67\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\">
  <path
    d=\\"M52.7761 50.1266L43.1742 28.4722C42.8202 27.674 42.6733 26.8028 42.7466 25.9359C42.82 25.069 43.1113 24.2331 43.5947 23.5026L55.9573 4.83683C56.5375 3.95996 57.3707 3.27058 58.3516 2.85566C59.3326 2.44075 60.4174 2.31889 61.4691 2.50547L83.8659 6.4689C84.7422 6.62436 85.5657 6.98867 86.2632 7.52951C86.9608 8.07035 87.5109 8.771 87.8648 9.56929L97.4667 31.2236M52.7761 50.1266C53.3465 51.4128 54.4166 52.4251 55.7512 52.9407C57.0857 53.4563 58.5753 53.4331 59.8922 52.876L94.6515 38.1738C95.9685 37.6167 97.005 36.5715 97.5329 35.2681C98.0608 33.9647 98.037 32.5099 97.4667 31.2236M52.7761 50.1266L64.6965 32.1259M97.4667 31.2236L75.8691 27.4002M64.6965 32.1259L43.099 28.3025M64.6965 32.1259L68.3441 32.7717C69.3959 32.9582 70.4807 32.8364 71.4616 32.4215C72.4426 32.0066 73.2757 31.3172 73.8559 30.4403L75.8691 27.4002M75.8691 27.4002L87.7895 9.39955\\"
    stroke=\\"#6D89B1\\"
    strokeWidth=\\"3\\"
    strokeLinecap=\\"round\\"
    strokeLinejoin=\\"round\\"
  />
  <path
    d=\\"M1.67397 61.647L6.36098 47.4957C6.53368 46.974 6.83555 46.502 7.23981 46.1216C7.64406 45.7412 8.13821 45.4641 8.67841 45.315L22.4852 41.5091C23.1335 41.3301 23.8216 41.3427 24.4627 41.5452C25.1037 41.7478 25.669 42.1312 26.0872 42.6472L34.9959 53.6297C35.3443 54.0596 35.5787 54.5668 35.6783 55.1064C35.7778 55.6459 35.7395 56.2012 35.5666 56.7228L30.8796 70.8741M1.67397 61.647C1.39556 62.4875 1.47045 63.4017 1.88216 64.1883C2.29387 64.975 3.00867 65.5697 3.86931 65.8416L26.5848 73.0183C27.4455 73.2902 28.3814 73.217 29.1869 72.8149C29.9923 72.4128 30.6012 71.7147 30.8796 70.8741M1.67397 61.647L14.988 57.976M30.8796 70.8741L22.2894 60.2828M14.988 57.976L6.39772 47.3847M14.988 57.976L16.4388 59.7648C16.8569 60.2807 17.4222 60.6642 18.0633 60.8667C18.7043 61.0693 19.3924 61.0818 20.0408 60.9028L22.2894 60.2828M22.2894 60.2828L35.6034 56.6119\\"
    stroke=\\"#6D89B1\\"
    strokeWidth=\\"3\\"
    strokeLinecap=\\"round\\"
    strokeLinejoin=\\"round\\"
  />
  <path
    d=\\"M95.2598 65.7402L98.9239 54.6774C99.0589 54.2696 99.2949 53.9006 99.6109 53.6032C99.927 53.3058 100.313 53.0892 100.736 52.9726L111.529 49.9974C112.036 49.8575 112.574 49.8673 113.075 50.0256C113.576 50.184 114.018 50.4837 114.345 50.8871L121.309 59.4727C121.582 59.8088 121.765 60.2053 121.843 60.6271C121.921 61.0489 121.891 61.483 121.756 61.8908L118.091 72.9536M95.2598 65.7402C95.0422 66.3974 95.1007 67.112 95.4226 67.727C95.7444 68.3419 96.3032 68.8068 96.976 69.0194L114.734 74.6298C115.407 74.8424 116.139 74.7852 116.768 74.4708C117.398 74.1565 117.874 73.6107 118.091 72.9536M95.2598 65.7402L105.668 62.8705M118.091 72.9536L111.376 64.6738M105.668 62.8705L98.9526 54.5907M105.668 62.8705L106.802 64.2688C107.129 64.6722 107.571 64.972 108.072 65.1303C108.573 65.2886 109.111 65.2984 109.618 65.1585L111.376 64.6738M111.376 64.6738L121.784 61.8041\\"
    stroke=\\"#6D89B1\\"
    strokeWidth=\\"3\\"
    strokeLinecap=\\"round\\"
    strokeLinejoin=\\"round\\"
  />
</svg>"
        xmlns="http://www.w3.org/2000/svg"
      >
        <RNSVGGroup
          fill={null}
          propList={
            Array [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M52.7761 50.1266L43.1742 28.4722C42.8202 27.674 42.6733 26.8028 42.7466 25.9359C42.82 25.069 43.1113 24.2331 43.5947 23.5026L55.9573 4.83683C56.5375 3.95996 57.3707 3.27058 58.3516 2.85566C59.3326 2.44075 60.4174 2.31889 61.4691 2.50547L83.8659 6.4689C84.7422 6.62436 85.5657 6.98867 86.2632 7.52951C86.9608 8.07035 87.5109 8.771 87.8648 9.56929L97.4667 31.2236M52.7761 50.1266C53.3465 51.4128 54.4166 52.4251 55.7512 52.9407C57.0857 53.4563 58.5753 53.4331 59.8922 52.876L94.6515 38.1738C95.9685 37.6167 97.005 36.5715 97.5329 35.2681C98.0608 33.9647 98.037 32.5099 97.4667 31.2236M52.7761 50.1266L64.6965 32.1259M97.4667 31.2236L75.8691 27.4002M64.6965 32.1259L43.099 28.3025M64.6965 32.1259L68.3441 32.7717C69.3959 32.9582 70.4807 32.8364 71.4616 32.4215C72.4426 32.0066 73.2757 31.3172 73.8559 30.4403L75.8691 27.4002M75.8691 27.4002L87.7895 9.39955"
            propList={
              Array [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={4285368753}
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth={3}
          />
          <RNSVGPath
            d="M1.67397 61.647L6.36098 47.4957C6.53368 46.974 6.83555 46.502 7.23981 46.1216C7.64406 45.7412 8.13821 45.4641 8.67841 45.315L22.4852 41.5091C23.1335 41.3301 23.8216 41.3427 24.4627 41.5452C25.1037 41.7478 25.669 42.1312 26.0872 42.6472L34.9959 53.6297C35.3443 54.0596 35.5787 54.5668 35.6783 55.1064C35.7778 55.6459 35.7395 56.2012 35.5666 56.7228L30.8796 70.8741M1.67397 61.647C1.39556 62.4875 1.47045 63.4017 1.88216 64.1883C2.29387 64.975 3.00867 65.5697 3.86931 65.8416L26.5848 73.0183C27.4455 73.2902 28.3814 73.217 29.1869 72.8149C29.9923 72.4128 30.6012 71.7147 30.8796 70.8741M1.67397 61.647L14.988 57.976M30.8796 70.8741L22.2894 60.2828M14.988 57.976L6.39772 47.3847M14.988 57.976L16.4388 59.7648C16.8569 60.2807 17.4222 60.6642 18.0633 60.8667C18.7043 61.0693 19.3924 61.0818 20.0408 60.9028L22.2894 60.2828M22.2894 60.2828L35.6034 56.6119"
            propList={
              Array [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={4285368753}
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth={3}
          />
          <RNSVGPath
            d="M95.2598 65.7402L98.9239 54.6774C99.0589 54.2696 99.2949 53.9006 99.6109 53.6032C99.927 53.3058 100.313 53.0892 100.736 52.9726L111.529 49.9974C112.036 49.8575 112.574 49.8673 113.075 50.0256C113.576 50.184 114.018 50.4837 114.345 50.8871L121.309 59.4727C121.582 59.8088 121.765 60.2053 121.843 60.6271C121.921 61.0489 121.891 61.483 121.756 61.8908L118.091 72.9536M95.2598 65.7402C95.0422 66.3974 95.1007 67.112 95.4226 67.727C95.7444 68.3419 96.3032 68.8068 96.976 69.0194L114.734 74.6298C115.407 74.8424 116.139 74.7852 116.768 74.4708C117.398 74.1565 117.874 73.6107 118.091 72.9536M95.2598 65.7402L105.668 62.8705M118.091 72.9536L111.376 64.6738M105.668 62.8705L98.9526 54.5907M105.668 62.8705L106.802 64.2688C107.129 64.6722 107.571 64.972 108.072 65.1303C108.573 65.2886 109.111 65.2984 109.618 65.1585L111.376 64.6738M111.376 64.6738L121.784 61.8041"
            propList={
              Array [
                "stroke",
                "strokeWidth",
                "strokeLinecap",
                "strokeLinejoin",
              ]
            }
            stroke={4285368753}
            strokeLinecap={1}
            strokeLinejoin={1}
            strokeWidth={3}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Poppins-Bold",
            },
            Object {
              "color": "#282828",
              "fontSize": 24,
            },
          ]
        }
      >
        Letters
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#6E757C",
              "fontSize": 18,
            },
            Object {
              "maxWidth": "60%",
            },
          ]
        }
      >
        2,000 words & photo on regular paper
      </Text>
    </TouchableOpacity>
  </View>
</View>
`;
